docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md

```diff
@@ -19,39 +19,39 @@ This page contains a list of available feature toggles. To learn how to turn on
+| `featureHighlights`                   | Highlight Grafana Enterprise features                                                                                                                                        |                    |
+| `cloudWatchDynamicLabels`             | Use dynamic labels instead of alias patterns in CloudWatch datasource                                                                                                        | Yes                |
+| `dataConnectionsConsole`              | Enables a new top-level page called Connections. This page is an experiment that provides a better experience when you install and configure data sources and other plugins. | Yes                |
+| `internationalization`                | Enables internationalization                                                                                                                                                 | Yes                |
+| `topnav`                              | Enables new top navigation and page layouts                                                                                                                                  | Yes                |
+| `cloudWatchCrossAccountQuerying`      | Enables cross-account querying in CloudWatch datasources                                                                                                                     | Yes                |
+| `newPanelChromeUI`                    | Show updated look and feel of grafana-ui PanelChrome: panel header, icons, and menu                                                                                          | Yes                |
+| `accessTokenExpirationCheck`          | Enable OAuth access_token expiration check and token refresh using the refresh_token                                                                                         |                    |
+| `emptyDashboardPage`                  | Enable the redesigned user interface of a dashboard page that includes no panels                                                                                             | Yes                |
+| `disablePrometheusExemplarSampling`   | Disable Prometheus exemplar sampling                                                                                                                                         |                    |
+| `logsSampleInExplore`                 | Enables access to the logs sample feature in Explore                                                                                                                         | Yes                |
+| `disableElasticsearchBackendQuerying` | Disable processing of queries and responses in Elasticsearch data source trough backend                                                                                      |                    |
```

Comment:
```suggestion
| `disableElasticsearchBackendQuerying` | Disable the processing of queries and responses in the Elasticsearch data source through backend                                                                                      |                    |
```

Reply:
@ivanahuckova , I'm unsure of what 'through backend' means. Can you please explain, and I can help with adjusting the words. Thank you!

Reply:
@chri2547 currently we are processing Elasticsearch queries and responses in typescript (on frontend). We are moving logic for processing queries and responses to data source backend in golang. This change should not be visible for users (we did extensive testing), but because processing of both - queries and responses is quite complex, there might be cases when processed response might have slight differences. In that case we recommend using this feature toggle to disable running queries torugh backend and we ask users to create a github issue to improve the experience and fix the issue.

Reply:
interesting - and thank you for the thorough description. How will a user know that this feature toggle is available and that they should create a GH issue to improve the experience?

Reply:
We will:

- Communicating this change via #elk channel in community slack
- Part of changelog

If you have any other suggestions, let me know!